src/paho/mqtt/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "1.4.0.dev0"
+__version__ = "1.4.1.dev0"
```

Comment:
This version should stay 1.4.0 as it's the future version number. I'll fix it after merging it.